pages/PreviewPage/[...picID].js

```diff
@@ -25,32 +25,58 @@ async function safeFavToCloud(picSRC) {
   return cloudinaryData;
 }
 
-async function safeFavToMongoDB(cloudinaryData, searchID) {
+async function safeFavToMongoDB(cloudinaryData) {
```

Comment:
```suggestion
async function saveFavToMongoDB(cloudinaryData) {
```

Reply:
I would also restrain from using implementation specific names (MongoDB) here, since tomorrow it could be another database, so I would keep it more generalised as such: `saveFavoriteToDB` or just `putFavorite`.